src/routes/memberships.rs

```diff
@@ -64,32 +62,25 @@ pub async fn create(
 
 pub async fn delete(
     conn: &mut Conn,
-    (current_user, db): (User, Db),
+    (actor, db): (PermissionsActor, Db),
 ) -> Result<impl Handler, Error> {
-    let membership_id = conn.param("membership_id").unwrap();
-    let membership_id = Uuid::parse_str(membership_id).map_err(|_| Error::NotFound)?;
+    let membership_id = conn
+        .param("membership_id")
+        .unwrap()
```

Comment:
Not sure about this unwrap. It'll panic if I leave out `membership_id` from the URL? I see other routes are shortcutting this with `?`.

Reply:
It will only panic if this code is called from a route that does not include a `:membership_id` segment. `memberships::delete` is only ever mounted at `/memberships/:membership_id`, which would not match at the router if the membership_id were missing

Reply:
I don't see a compelling reason to map the None to an Error that will categorically never be constructed. I'd love to figure out how to use the type system to make the unwrap unnecessary, since we can be sure that "membership_id" exists but no other param does, but that would require tracking around some awkward generics

Reply:
Ah OK, my mistake I thought it was a URL parameter with `&`. No need for any typing changes, but could consider having the unwrap be `.expect("membership_id should have been set by the route")` or something to that tune.

Reply:
As this is unreachable code, unwrap seems appropriate here. We have exactly [one](https://github.com/divviup/divviup-api/blob/main/src/db.rs#L14) `expect` use in src, and that probably should be switched to an unwrap for consistency, whereas we have 61 uses of unwrap. An unwrap panic contains the backtrace since we compile with unwinding, which should be sufficient for investigating any bugs involving use of unwrap in any reachable code path.